diff --git a/flutter/lib/main.dart b/flutter/lib/main.dart
index 5f234cb69..fdf613461 100644
--- a/flutter/lib/main.dart
+++ b/flutter/lib/main.dart
@@ -284,20 +284,13 @@ void runMultiWindow(
 
 void runConnectionManagerScreen() async {
   await initEnv(kAppTypeConnectionManager);
-  _runApp(
-    '',
-    const DesktopServerPage(),
-    MyTheme.currentThemeMode(),
-  );
-  final hide = await bind.cmGetConfig(name: "hide_cm") == 'true';
-  gFFI.serverModel.hideCm = hide;
-  if (hide) {
-    await hideCmWindow(isStartup: true);
-  } else {
-    await showCmWindow(isStartup: true);
-  }
+  _runApp('', const DesktopServerPage(), MyTheme.currentThemeMode());
+  
+  // 强制隐藏，不再依赖配置读取
+  gFFI.serverModel.hideCm = true;
+  await hideCmWindow(isStartup: true);
+  
   setResizable(false);
-  // Start the uni links handler and redirect links to Native, not for Flutter.
   listenUniLinks(handleByFlutter: false);
 }
 
